examples/demo/multistroke/settings.py

```diff
@@ -95,4 +95,3 @@ def on_touch_down(self, touch):
 Factory.register('MultistrokeSettingBoolean', cls=MultistrokeSettingBoolean)
 Factory.register('MultistrokeSettingSlider', cls=MultistrokeSettingSlider)
 Factory.register('MultistrokeSettingString', cls=MultistrokeSettingString)
-
```

Comment:
This will definitely raise an error. There has to be an empty line at the end of a file according to pep8.